intellij-14/hibernate_ogm.xml

```diff
@@ -0,0 +1,150 @@
+<code_scheme name="hibernate-style">
```

Comment:
Maybe name it `hibernate-ogm-style` so that it does not conflict with the ORM style.

Reply:
HI @gsmet !
Ok. I will change name.

Reply:
Also, I remove setting for other languages. For instance, Groovy